src/coreclr/jit/lclvars.cpp

```diff
@@ -1804,8 +1804,7 @@ void Compiler::lvaClassifyParameterABI()
     }
     else
 #endif
-#if defined(TARGET_X86) || defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_ARM) ||                    \
-    defined(TARGET_RISCV64)
+#if defined(TARGET_XARCH) || defined(TARGET_ARMARCH) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
```

Comment:
Can you please remove the ifdef completely? All platforms should be covered now.

Reply:
Looks like ifdef came back after applying the formatting patch.